silnlp/common/corpus.py

```diff
@@ -135,11 +138,17 @@ def exclude_books(corpus: pd.DataFrame, books: Set[int]) -> pd.DataFrame:
 
 
 def get_terms_metadata_path(list_name: str) -> Path:
-    return MT_TERMS_DIR / f"{list_name}-metadata.txt"
+    terms_dir = MT_TERMS_DIR / f"{list_name}-metadata.txt"
+    if not terms_dir.exists():
```

Comment:
I would use `is_file()` instead of `exists()`. This method checks both that the path exists and that it is a file.

Reply:
Yes, thanks that's much better.  I think there's an is_dir()  for folders too.